eng/tools/suppressions/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
```

Comment:
It seems `outDir` must be set in the package dir itself.  It cannot be inherited since it's resolved relative to the `tsconfig.json` where it's defined, not the file extending it.